recommence/Checkpoint.py

```diff
@@ -40,7 +51,15 @@ def remove(self) -> None:
 
     def _load_if_exists(self) -> None:
         if os.path.exists(self.save_path):
-            if os.path.exists(self._data_path):
-                with open(self._data_path, 'rb') as f:
-                    self._data = pickle.load(f)
+            try:
+                if os.path.exists(self._data_path):
+                    try:
+                        with open(self._data_path, 'rb') as f:
+                            self._data = pickle.load(f)
```

Comment:
Normally, I'd complain about the level of nesting here (i.e. the number of indentations), but I have a PR in progress that will flatten this out so we'll deal with it in another PR